sdks/python/apache_beam/options/pipeline_options_test.py

```diff
@@ -192,47 +192,53 @@ def _add_argparse_args(cls, parser):
     options = PipelineOptions(['--redefined_flag'])
     self.assertTrue(options.get_all_options()['redefined_flag'])
 
+  # TODO(mariagh): Require unique names only within a test.
```

Comment:
So you mean still numbering them but by file? So my idea is to number them in absolute terms and you propose file1_vp1, file1_vp2... file2_vp1, file2_vp2...?